pkg/models/temp_user.go

```diff
@@ -35,8 +36,8 @@ type TempUser struct {
 	Code        string
 	RemoteAddr  string
 
-	Created time.Time
-	Updated time.Time
+	Created int64
```

Comment:
Why is it necessary to change the type?

Reply:
Please see thread https://github.com/grafana/grafana/pull/27361#pullrequestreview-488743815